components/observatorium.libsonnet

```diff
@@ -224,7 +224,7 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
       namespace: obs.config.namespace,
       replicas: 1,
       commonLabels+:: obs.config.commonLabels,
-      logs: {
+      [if std.length(obs.config.loki) != 0 then 'logs']: {
```

Comment:
Consider using the style in [L315-317](https://github.com/observatorium/deployments/pull/316/files#diff-bf6438e7ce24362188f3a78b9183b071R315-R317) for adding optional into a config object.

Reply:
Done